doc/website/getting-started/overview.md

```diff
@@ -58,17 +58,16 @@ and exhaust memory. We have to handle this potential error since the expected cl
 attached. This means we get a warning (or an error when build in strict mode) when we don't handle it. We could
 also explicitly discard it with `IOX_DISCARD_RESULT` which is discouraged. If you want to know more about
 `expected`, take a look at
-[How optional and error values are returned in iceoryx](how-optional-and-error-values-are-returned-in-iceoryx.md).
-
+[How optional and error values are returned in iceoryx](../concepts/how-optional-and-error-values-are-returned-in-iceoryx.md).
 Let's create a corresponding subscriber.
 
 ```cpp
 iox::popo::Subscriber<CounterTopic> subscriber({"Group", "Instance", "CounterTopic"});
 ```
 
 Now we can use the subscriber to receive data. For simplicity, we assume that we periodically check for new data. It
-is also possible to explicitly wait for data using the [WaitSet](waitset.md) or the [Listener](callbacks.md). The
-code to receive the data is the same, the only difference is the way we wake up before checking for data.
+is also possible to explicitly wait for data using the [WaitSet](../../../iceoryx_examples/waitset/README.md) or 
```

Comment:
Not sure here if we should link to the website (../examples/waitset.md) or to github. But it should be consistent which is not the case at the moment. In doc/website/concepts/qos-policies.md for instance, we link to the "website" iceoptions readme.

Reply:
I fixed this in the `qos-policies.md` file. For now I would consistently link to the github examples since the example markdown website file is more or less empty.

For github users it would feel like a dead end and a broken link

Reply:
Okay with me. I didn't find other links to `../examples/`.